formats/taxon.jsonnet

```diff
@@ -24,6 +24,13 @@
     associated_taxons: "A list of associated taxons whose children should be included as children of this taxon",
   },
   links: (import "shared/base_links.jsonnet") + {
-    parent_taxons: "The list of taxon parents (DEPRECATED: use the edition links instead)",
+    parent_taxons: {
+      "$ref": "#/definitions/guid_list",
+      description: "The list of taxon parents (DEPRECATED: use the edition links instead)",
```

Comment:
I went back in time, and found the source of this deprecation message ( c31ba3a8aea1b6f958ffde8552c845634e60f353 ).

I think the reasoning makes sense, and this might be a sensible time to switch Content Tagger to use the edition links, as we are making a change relating to the root_taxon link type?

Reply:
I think that got abandoned because there was something with the publishing-api not doing reverse links for edition links.